src/vs/workbench/api/common/extHostTerminalService.ts

```
@@ -225,6 +225,10 @@ export class ExtHostPseudoterminal implements ITerminalChildProcess {
 		}
 	}
 
+	processBinary(data: string) {
+		throw new Error('not implemented');
+	}
```

Let's fail silently here with a comment